scripts/households.py

```diff
@@ -56,19 +55,15 @@ def mu_labor(n, chi, *args, b_sp1):
 
     '''
 
-    theta = 0.9  # frisch elasticity of labor
-    l_tilde = 1.0  # Maximum labor supply
-
-    b, nu = elliptical_u_est.estimation(theta, l_tilde)
     beta, sigma, r, w, n, b_init = args
```

Comment:
@jtregde Notice how you have some parameters in list of arguments in line 41 and another set packed into the `args` tuple passed as an argument to the function.  

You might want to be consistent so that you are doing something similar across functions.

One suggestion, for a function like this, pack the parameters in a tuple and leave other variables are separate arguments to the function.  e.g.,
```
def mu_labor(n, b_sp1, params):
...
    beta, sigma, chi, l_tilde, b_ellip, upsilon = params
...
```